### DISCUSSION TITLE
Map won't show - Valetudo/Home Assistant

### DISCUSSION THREAD

**Discussion by dnl0319** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#361] [Q&A]
Hello everyone,

I was trying to install the xiaomi vaccuum map card but couldn't get the vacuum map working.

![grafik](https://user-images.githubusercontent.com/61114433/155519694-27233c09-1be4-4f49-88d7-87c6a717d5cb.png)

This is the result.

My vacuum hast valetudo 2022.02.0 isntalled.

I also have Lovelace valetudo map card installed and it's working:

![grafik](https://user-images.githubusercontent.com/61114433/155519903-2351a3c2-b677-49b0-a5f1-e25baadf2b17.png)

But it still won't work.

I also looked at the xiaomi map extractor. But there was written you need a ussername and a password, and it looks like for this is for the xiaomi account. Which I don't have.

Is there a way to get this working?

**Comment by PiotrMachowski** [comment]
Lovelace valetudo map card retrieves raw map data from mqtt, here you need an already parsed image. You can use [ICantBelieveItsNotValetudo](https://github.com/Hypfer/ICantBelieveItsNotValetudo) to do it.
